### PR TITLE
Fixed angular version issues

### LINKED ISSUE #278: Angular version issues
The angular version specified in the README.md file and in the app's signup and login component doesn't match the one actually used in the project.

### PR DESCRIPTION
**Pull request by OHassini**
Check by changing each `[ ]` to `[x]` Please take note of the white-space as it matters.

- This Pull request
   - [ ] Fixes a bug.
   - [x] Implements a new feature.
- [x] I have read the **[CONTRIBUTING.md]** .

- [x] I have included a description of changes below.
- [ ] I have included screenshots  showing before and after the changes were made.
- [x] I have squashed the commits **(mandatory)**.

## Description / Changes

- Fixes #278 
- I have updates the angular version in the README.md file to match the one used in the project.
- I have also implemented a feature that automatically updates the angular version specified in the app's login and signup component when the project's angular version changes.

### CODE REVIEW

**Comment by KeenWarrior** [comment]
@OussEmaDevCode Can you please correct the build failure.

**Comment by OHassini** [comment]
@KeenWarrior done

File: src/app/core/login/login.component.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
       </div>
       <div class="section">
         <div class="section-title">Development Stack</div>
-        Angular 6 for Front-End<br>
+        Angular {{angularVersion}} for Front-End<br>
```
Comment (OHassini) [inline src/app/core/login/login.component.html:14]: Angular {{angularVersion}} for Front-End angular<br>

File: src/app/core/login/login.component.ts [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { AuthenticationService } from '../../services/authentication.service';
 import { Router } from '@angular/router';
 import { User } from '../../services/user.service';
+import {VERSION} from '@angular/core';
```
Comment (OHassini) [inline src/app/core/login/login.component.ts:6]: import { VERSION } from '@angular/core';

File: src/app/core/login/login.component.ts [hunk 3]
```diff
@@ -11,6 +12,7 @@ import { User } from '../../services/user.service';
 })
 export class LoginComponent implements OnInit {
 
+  angularVersion = VERSION.major;
```
Comment (OHassini) [inline src/app/core/login/login.component.ts:15]: angularVersion  = VERSION.major;

File: .travis.yml [hunk 4]
```diff
@@ -18,7 +21,6 @@ before_script:
   - npm install -g @angular/cli
   - export CHROME_BIN=chromium-browser
   - export DISPLAY=:99.0
```
Comment (OHassini) [inline .travis.yml:23]: - export CHROME_BIN = chromium-browser

**Pull request closed without merging**